### PR TITLE
add orange labs node

### PR DESCRIPTION
**Pull request by pascallimeux**
# Hi! Thank you for submitting a Pull Request!

## If this is not for an enode you can ignore this and delete it.

ENODE: **Replace this with your enode with format (enode://hash@host:2100?discport=0)**

### CODE REVIEW

**Comment by pascallimeux** [comment]
add Orange node

**Comment by alejandroalffer** [comment]
Update bootnode in DigitelTS, but still not syncing. Ports tcp/udp/21000 returns connection refused from the new node. Keep in touch!

**Comment by pascallimeux** [comment]
Hello Alejandro,
I don't understand because the docker container works and the ports are open.
do you know what checks I can do?
thanks

**Comment by alejandroalffer** [comment]
Hi!

A full-nat connection its need for `geth` process (well... in SOHO geth will try to open a uPNP connection in the router).

But first, let's see how its working the processes in the container:

```sh
ps auxww|grep geth
```

... and search for the `--port 21000` argument 

Also, if the node it's running in Docker, it's need to make a port mapping for the procesess:

```sh
docker ps -a
```

... and look for a ```0.0.0.0:21000->21000/tcp, 0.0.0.0:21000->21000/udp``` in the PORT columns.

If these params are ok, it could be a network problem: be sure that 21000/tcp and 21000/udp are open for DockerHost/VM machine.

Keep in touch!

**Comment by pascallimeux** [comment]
Hi,
thanks for all this info:
ps auxww|grep geth in container does not give answer
docker ps returns the ports mapper correctly
cca23067aced alastria/alastria-node-general "entrypoint.sh /bin/..."   7 hours ago Up 7 hours 0.0.0.0:80->80/tcp, 0.0.0.0:443->443/tcp, 0.0.0.0:8443->8443/tcp, 0.0.0.0:9000->9000/tcp, 0. 0.0.0:21000->21000/tcp, 0.0.0.0:22000->22000/tcp, 0.0.0.0:21000->21000/udp REG_Orange_Labs_T_2_8_00

the ports are open I checked with nmap from my Mac 
nmap -p 21000 90.84.193.56

in conclusion geth does not work on the docker container, but I don't know why, because I only run ./init.sh like describes on documentation

**Comment by alejandroalffer** [comment]
Definitely, `geth` is not running on your system.

Please, run a `docker logs REG_Orange_Labs_T_2_8_00` to see the boot process: maybe you will find the reason there. Anyway, log into the container by `docker exec -it REG_Orange_Labs_T_2_8_00 /bin/bash` and try to start geth with start.sh

Best regards!

PD. How did you get the end of your container? It means that the process was running at some point.

**Comment by pascallimeux** [comment]
Hi Alejandro,
when I try to launch start.sh I have the following error: ./start.sh: line 70: /home/cloud/alastria/data/IDENTITY: No such file or directory.
when I try to start get in the docker container, I have the following error: Fatal: Consensus not specified. Exiting!!
and the container log end with:
WARN [03-29|10:07:24.195] Failed to decode stats server message    err="json: cannot unmarshal string into Go value of type map[string][]interface {}"
WARN [03-29|10:07:38.451] Full stats report failed                 err="write tcp 172.17.0.2:40700->185.180.8.152:80: use of closed network connection"
WARN [03-29|10:07:54.196] Failed to decode stats server message    err="json: cannot unmarshal string into Go value of type map[string][]interface {}"
INFO [03-29|10:08:00.222] Got interrupt, shutting down... 
INFO [03-29|10:08:00.222] HTTP endpoint closed                     url=http://127.0.0.1:22000
INFO [03-29|10:08:00.222] IPC endpoint closed                      endpoint=/root/alastria/data/geth.ipc
INFO [03-29|10:08:00.222] Stats daemon stopped 
INFO [03-29|10:08:00.222] Blockchain manager stopped 
INFO [03-29|10:08:00.222] Stopping Ethereum protocol 
INFO [03-29|10:08:00.222] Ethereum protocol stopped 
INFO [03-29|10:08:00.222] Transaction pool stopped 
INFO [03-29|10:08:00.222] Database closed                          database=/root/alastria/data/geth/chaindata

best regards

**Comment by alejandroalffer** [comment]
The installation is not working. In fact, the enode that you indicate in the Pull Request is not possible that it has been generated correctly: I will reverse the commit until you have the active node.

Errors seems two things:
* the first error is that the "IDENTITY" file must exist in the indicated path, and must contain the name of the node
* the second incident is that the database has not been started with the correct genesis file

I invite you to try the installation through this repository, https://github.com/alastria/alastria-node-quorum and do the installation through docker-compose way

On the other hand, send the PR but do not accept it: someone from the Alastria team will validate your installation before adding you to the network

**Comment by pascallimeux** [comment]
ok Thank you Alejandro, I will try to install the node from the address you gave me
but what does it mean "send the PR but do not accept it". 
the first time I had used this tutorial to install the node:
https://medium.com/babel-go2chain/setting-in-motion-a-regular-node-in-the-telsius-network-of-alastria-c2d67b8369c7

best 
Pascal

**Comment by pascallimeux** [comment]
Hi Alejandro,
I made the installation as you told me and here is my PR I think:
enode://c5f9010783e37ebf6ac840e9549f0ce2ff6594ecd383ecb51b406f11538ebe954ffcde8fe3ef5080b27b416a6a622a387a1f266c9e01fe2dc14f88d7dfea1724@90.84.193.56:21000?discport=0
I checked geth works in the container

best

Pascal

**Comment by alejandroalffer** [comment]
Seems much better! I'll add the new node in a hurry

**Comment by pascallimeux** [comment]
thanks Alejandro

**Comment by alejandroalffer** [comment]
Finished! Node added in DigitelTS bootnode
